MAUI/Rating/Restrict-User-Selection.md

```diff
@@ -0,0 +1,34 @@
+---
+
+layout: post
+title: Restrict user Selection in .NET MAUI Rating control | Syncfusion
+description: Learn here all about Restrict User Selection support in Syncfusion .NET MAUI Rating (SfRating) control and more.
+platform: maui
+control: Rating
+documentation: ug
+
+---
+
+# Restrict user selection in .NET MAUI Rating (SfRating)
+
+[`SfRating`] control provides support for changeable or unchangeable values for Rating control. This is achieved by the [`IsReadOnly`] property. When this property is set to True, the Rating value becomes unchangeable. By default, this property value is set to False.
```

Comment:
[`SfRating`] control supports changeable or unchangeable values for Rating control.

Reply:
Changed